### PR TITLE
Gravity update to bird.js 

### PR DESCRIPTION
**Pull request by buensons**
Hello everyone!

First off, I would like to say that I'm new to GitHub and this is my first pull request, so if I did something wrong, I'm sorry.

Recently, I was inspired by Coding Train's Flappy Bird video to make my own version of the game in Java. As I created my game, I would like to add one minor change to this JavaScript version, which I think improves it a little bit, as it adds acceleration of gravity to the game. Before, the 'bird' was going down with a constant speed, and after my changes it accelerates while going downwards.

**Changes:**

1. I removed the 'this.lift' variable and replaced it with this.delay which is used to limit the speed of going up, by setting a delay.

2. Changed up() function to implement 'delay'

3. Removed 'this.velocity *= 0.9', because it's function was to limit the speed, and as I said I did it by implementing delay. It was also causing the lack of acceleration.

Hopefully you like my change, and my contribution is somewhat useful. Once again, it's my first time contributing to a project on GitHub, so I'm sorry for any mistakes. 

Cheers!

### CODE REVIEW

File: bird.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ class Bird {
     this.x = 64;
 
     this.gravity = 0.6;
-    this.lift = -15;
+    this.delay = 0;
```
Comment (shiffman) [inline bird.js]: Would a `boolean` flag variable make more sense here?

Comment (buensons) [reply bird.js]: Yes, you are right. A boolean variable would work, and it's going ot be easier to read. Can I correct it by making another pull request?

File: bird.js [hunk 2]
```diff
@@ -26,12 +26,17 @@ class Bird {
   }
 
   up() {
-    this.velocity += this.lift;
+    if (this.delay)
```
Comment (ZwergB) [inline bird.js]: Isn't this whole `this.delay` variable kind of irrelevant? Sorry if  I miss something but it seems like you could delete the whole if-statement and just leave line 33.

File: bird.js [hunk 3]
```diff
@@ -26,12 +26,17 @@ class Bird {
...
+        this.delay = false;
+
+    if (!this.delay) {
+        this.velocity = -7;
```
Comment (ZwergB) [inline bird.js]: Creating a global variable for the `-7` would be a good thing to do. It's easier to modify static values when they are global.

**Comment by buensons** [comment]
Yes, you're right. I can see now that this.delay is completely useless here. I used it in a bit different way in Java to fix one problem, but I can see that it doesn't make sense here, and just the line 33 will give the same results.

**Comment by shiffman** [comment]
Thank you for this contribution! I am going to merge it and test it out. I might revert if I prefer the previous behavior but very thankful for this suggestion. More soon!

**Pull request merged**